gensim/models/ensemblelda.py

```diff
@@ -98,6 +98,8 @@
 
 """
 
+print('asdf')
```

Comment:
?

Reply:
it was removed in 766f562a065f08b3dfdc0754e138ea2a15e38090

I added it to see if the correct module was imported during development